tools/envoy_headersplit/headersplit_test.py

```diff
@@ -0,0 +1,159 @@
+# Lint as: python3
+"""Tests for headersplit."""
+
+import unittest
+import headersplit
+import io
+import clang.cindex
+from clang.cindex import TranslationUnit, Index, CursorKind
+clang.cindex.Config.set_library_path("/opt/llvm/lib")
+
+
+class HeadersplitTest(unittest.TestCase):
+  source_code_hello_world = """
```

Comment:
Can these constants be declared in the test cases that use them? It's difficult for the reader to have to scroll around looking for the string constant up here.